content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -16,6 +16,20 @@ weight: 1
 
 # Corda Enterprise Edition 4.9 release notes
 
+## Corda Enterprise Edition 4.9.5 release notes
+
+Corda Enterprise Edition 4.9.5 is a patch release of Corda Enterprise focused on resolving issues.
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and for the latest upgrade guide, see [Upgrading a CorDapp or node](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+### Fixed issues
+
+* Previously, a memory leak in the transaction cache occurred due to weight of in-flight entries being undervalued. Improvements to prevent in-flight entry weights from being undervalued and estimated more correctly means a large decrease in the total size of cached entities. 
```

Comment:
```suggestion
* Previously, a memory leak in the transaction cache occurred due to the weight of in-flight entries being undervalued. Improvements have been made to prevent in-flight entry weights from being undervalued and because they are now estimated more correctly, this results in a large decrease in the total size of cached entities.
```

Reply:
Done.